### PR TITLE
docs: update rspack-start

### PR DESCRIPTION
**Pull request by 9aoy**
## Types of changes

- [ ] Docs change / Dependency upgrade
- [ ] Bug fix
- [ ] New feature / Improvement
- [ ] Refactoring
- [ ] Breaking change

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx [hunk 1]
```diff
@@ -15,14 +15,14 @@ Of course, you can also access it manually by following the steps below:
 1. Install rspack-provider（The version needs to be the same as the modern.js version).
 
 ```
-pnpm add @modern-js/builder-rspack-provider@2.4.0 -D
+pnpm add @modern-js/builder-rspack-provider@2.8.0 -D
```
Comment (chenjiahan) [inline packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx:18]: How about remove the fixed version

File: packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx [hunk 2]
```diff
@@ -59,10 +59,14 @@ const provider = builderRspackProvider({
 
 Builder aims to eliminate the main differences between different bundlers and help users migrate at a lower cost. However, due to the different capabilities of each bundler, users still need to be aware of some differences.
 
+This article will introduce the differences between webpack and rspack from the perspective of Builder. If you need to know more in-depth differences, please refer to the [rspack documentation](http://www.rspack.org/guide/migrate-from-webpack.html).
```
Comment (chenjiahan) [inline packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx:62]: `rspack` should always be 'Rspack'

File: packages/builder/builder-doc/docs/zh/guide/advanced/rspack-start.mdx [hunk 3]
```diff
@@ -165,3 +168,12 @@ export default {
 ```
 
 关于 rspack 的详细配置信息，请参考 [rspack 官网](https://rspack.org/)。
+
+### 更灵活的修改配置方式
+
+暂未提供类似 webpack-chain 方式来灵活修改 rspack config 的方法。
```
Comment (chenjiahan) [inline packages/builder/builder-doc/docs/zh/guide/advanced/rspack-start.mdx:174]: 缺少主语，「Builder 暂未提供 xxx」

File: packages/builder/builder-doc/docs/zh/guide/advanced/rspack-start.mdx [hunk 4]
```diff
@@ -165,3 +168,12 @@ export default {
...
+
+如需增删插件 / 规则，可使用 [prependPlugins](/api/config-tools.html#prependplugins)、[addRules](/api/config-tools.html#addrules) 等工具方法实现。
+
+如需对已有插件/规则进行修改，推荐优先使用 builder 已有配置项进行修改。
```
Comment (chenjiahan) [inline packages/builder/builder-doc/docs/zh/guide/advanced/rspack-start.mdx:178]: builder -> Builder

File: packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx (hunk 1 above)
Comment (9aoy) [reply packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx:18]: This is not necessary, but I think it is better to update to a newer version

File: packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx (hunk 2 above)
Comment (9aoy) [reply packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx:62]: ......

File: packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx (hunk 1 above)
Comment (chenjiahan) [reply packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx:18]: Why not just remove the version?

Comment (9aoy) [reply packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx:18]: ok

**Review by chenjiahan — DISMISSED** [review]

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**